cluster/ci/config/prow/config.yaml

```diff
@@ -1732,3 +1732,13 @@ periodics:
   labels:
       master: ci.openshift.redhat.com
   interval: 2h
+- name: azure_build_base_image
+  agent: jenkins
+  labels:
+      master: ci.openshift.redhat.com
+  interval: 24h
```

Comment:
It should be enough to build the base image weekly, it makes no sense to build it daily, also we don't have any clean up in place yet.. So it is possible to set this to 1w or 7d or 168h?